pkg/apis/knative/v1alpha1/deployer_types.go

```diff
@@ -64,6 +66,11 @@ const (
 	IngressPolicyExternal     IngressPolicy = "External"
 )
 
+type Scale struct {
+	Minimum *int64 `json:"minimum,omitempty"`
+	Maximum *int64 `json:"maximum,omitempty"`
```

Comment:
int64 may be a wee bit optimistic about our ability to scale

Reply:
still being optimistic with an `*int32`, but matching knative's type for Desired/Actual Scale